ddl/ddl.go

```diff
@@ -1618,7 +1621,7 @@ func processAllJobs(process func(*sess.Session, *model.Job, model.AdminCommandOp
 	for {
 		var jobs []*model.Job
 		jobs, err = getJobsBySQL(ns, JobTable,
-			fmt.Sprintf("job_id >= %s order by job_id asc limit %s",
```

Comment:
Why need to add `processing` ? How about the job is queuing?

Reply:
good one, remove it.